### PR TITLE
Add initial Dockerfile

### PR DESCRIPTION
**Pull request by gliptak**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

After this merged, I will update the docs

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM python:3.8-alpine
```
Comment (kddejong) [inline Dockerfile]: Validating this part internally for software licensing.

Comment (gliptak) [reply Dockerfile]: Alpine for smaller size. Could use ```3.8-slim``` (Debian Buster) instead

**Comment by kddejong** [comment]
weird error... I think this may be a misconfiguration with our linting.

**Comment by gliptak** [comment]
@kddejong current ```astroid``` doesn't have ```Python 2``` support

https://pypi.org/project/astroid/
https://travis-ci.org/PyCQA/astroid

**Comment by gliptak** [comment]
Same for https://pypi.org/project/pylint/

**Comment by gliptak** [comment]
@kddejong after rebasing PR build is green

**Pull request merged**